kube-enforcer/values.yaml

```diff
@@ -85,7 +85,7 @@ microEnforcerImage:
 # KubeBench Image
 kubebenchImage:
   repository: "aquasec/kube-bench"  # Default aqua registry KubeBench repository name
-  tag: "v0.6.15"
+  tag: "v.0.7.0"
```

Comment:
tag is having extra dot[.], it should be this `v0.7.0`

Reply:
fixed